custom/enikshay/integrations/bets/repeaters.py

```diff
@@ -242,3 +250,52 @@ def create_case_repeat_records(sender, case, **kwargs):
     create_repeat_records(BETSAYUSHReferralRepeater, case)
 
 case_post_save.connect(create_case_repeat_records, CommCareCaseSQL)
+
+
+class BETSUserRepeater(Repeater):
```

Comment:
Any reason not to make these part of the product? Not sure if that is something other people might find useful.